testsuite/tests/multicluster/load_balanced/test_unsupported_geocode.py

```diff
@@ -0,0 +1,15 @@
+"""Test not supported geocode in geo load-balancing"""
+
+import pytest
+
+from testsuite.kuadrant.policy import has_condition
+
+pytestmark = [pytest.mark.multicluster]
+
+
+def test_unsupported_geocode(dns_policy):
+    """Change default geocode to not existent one and verify that policy became not enforced"""
+    dns_policy.model.spec.loadBalancing.geo.defaultGeo = "XX"
+    dns_policy.apply()
+
+    assert dns_policy.wait_until(has_condition("Enforced", "False"))
```

Comment:
I think that it would be cool if more specific assert is here. Once your https://github.com/Kuadrant/testsuite/pull/520 is merged you can use great approach you used there. Something along the lines of
```
assert dns_policy.wait_until(
        has_record_condition("Ready", "False", "ProviderError", "Cannot find location")
    ), f"DNSPolicy did not reach expected record status, instead it was: {dns_policy.model.status.recordConditions}"
```